documentation/src/main/markdown/packagemgrs/opam.md

```diff
@@ -21,8 +21,9 @@ exe to be present on your $PATH. You can also override the location for `opam` e
 The OPAM Build Detector will work in the following way on your project:
 
 1. [detect_product_short] OPAM Build Detector will run `opam --version` to get the version of opam on your machine.
-2. If the version of opam is greater than or equal to 2.2.0, then [detect_product_short] would run `opam tree . --with-test --with-doc --with-dev --recursive`
-to get the list of resolved packages installed in the current [switch](https://ocaml.org/docs/opam-switch-introduction#opam-switch-introduction) for the project.
+2. If the version of opam is greater than or equal to 2.2.0, then [detect_product_short] will run the command `opam tree . --with-test --with-doc --with-dev --recursive --json=JSON_FILE_OUTPUT_PATH` if the version of opam is equal to, or greater than 
```

Comment:
I think this first part of the sentence can be removed now: "If the version of opam is greater than or equal to 2.2.0, then"

Reply:
Totally missed it.